### ISSUE TITLE
support `SHOW CREATE SOURCE` command

### ISSUE THREAD

**Issue by neverchanje** [risingwavelabs/risingwave#9047]
### Is your feature request related to a problem? Please describe.

# RisingWave `SHOW CREATE SOURCE` Command

RisingWave's `SHOW CREATE SOURCE` command is used to display the SQL
statement that was used to create a particular source in the current
schema.

The basic syntax for the `SHOW CREATE SOURCE` command is as follows:

``` sql
SHOW CREATE SOURCE source_name;
```

Here is a brief explanation of each element:

  - `source_name`: This is the name of the source for which you want to
    display the SQL statement.

If you execute the `SHOW CREATE SOURCE` command with a source name,
RisingWave will display the SQL statement that was used to create that
source. The output will include information about the source’s name,
type, connection properties, format, and any other relevant options.

Here is an example of how to use the `SHOW CREATE SOURCE` command to
display the SQL statement for a source named `orders`:

``` sql
SHOW CREATE SOURCE orders;
```

This command will display the SQL statement that was used to create the
`orders` source, including all of its configuration options.

In summary, the `SHOW CREATE SOURCE` command is a useful tool for
understanding how sources are configured in a RisingWave application. By
using this command, you can verify the settings for a particular source
and troubleshoot any issues that may arise.
